cdap-ui/app/cdap/components/FieldLevelLineage/FieldLevelLineage.scss

```diff
@@ -18,6 +18,8 @@
   .time-picker-dropdown {
     margin-left: 15px;
     margin-top: 15px;
+
+    .dropdown-menu.show { display: block; }
```

Comment:
Instead of just fixing this instance, it might be better to apply a global fix, so that future instances where we use `ButtonDropdown` won't have to do this.

In `bootstrap_4_patch.scss`, Ajai added this fix:

`.cdap-body-container .dropdown .dropdown-menu[aria-hidden="false"] { display: block; }`

However it didn't apply here because `ButtonDropdown` doesn't add a top-level `.dropdown` class. Maybe we can modify this to just be:

`.cdap-body-container .dropdown-menu[aria-hidden="false"] { display: block; }`

Reply:
great find!